From d744514d84f24319e6283ce48e675958c34822b8 Mon Sep 17 00:00:00 2001
From: Atul Raut <araut@codeaurora.org>
Date: Fri, 15 Jun 2018 12:19:25 -0700
Subject: [PATCH] defconfig: msm8996: Enable NTB for Perf build

Enable perf build support for NTB framework and
IDT hardware driver.

Change-Id: I5b9bf0d379fda31954fc605a22579e2c48cc0df5
Signed-off-by: Atul Raut <araut@codeaurora.org>
---
 arch/arm64/configs/msm-auto-perf_defconfig | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/arch/arm64/configs/msm-auto-perf_defconfig b/arch/arm64/configs/msm-auto-perf_defconfig
index 77f186f..5a77463 100644
--- a/arch/arm64/configs/msm-auto-perf_defconfig
+++ b/arch/arm64/configs/msm-auto-perf_defconfig
@@ -636,3 +636,10 @@ CONFIG_CRYPTO_AES_ARM64_CE_BLK=y
 CONFIG_CRYPTO_AES_ARM64_NEON_BLK=y
 CONFIG_CRYPTO_CRC32_ARM64=y
 CONFIG_QMI_ENCDEC=y
+CONFIG_NTB_NETDEV=y
+CONFIG_NTB=y
+CONFIG_NTB_IDT=y
+CONFIG_NTB_PINGPONG=m
+CONFIG_NTB_TOOL=m
+CONFIG_NTB_TRANSPORT=y
+CONFIG_NTB_PERF=m
--
